src/model/GroundHeatExchangerVertical.cpp

```diff
@@ -443,6 +463,7 @@ namespace model {
     OS_ASSERT(getImpl<detail::GroundHeatExchangerVertical_Impl>());
 
     setNumberofBoreHoles(120);
+    setBoreHoleTopDepth(1);
```

Comment:
harcoded in ctor already!

Reply:
What? Where?

Reply:
(I was just saying that this already sets it in the ctor, so making it required makes sense)